packages/dialtone/lib/build/less/components/table.less

```diff
@@ -43,7 +43,7 @@
 
     // Table Head Styles
     thead {
-        border-bottom: var(--dt-size-200) solid var(--table-color-border);
+        border-bottom: var(--dt-size-100) solid var(--table-color-border);
```

Comment:
What's this about? This is a global change to the Table style and should probably remain `200`

Reply:
Oh my bad! Just fixed it in the new commit.